Cargo.toml

```diff
@@ -24,6 +24,7 @@ log = "0.3"
 env_logger = "0.3"
 toml = { version = "0.2", features = ["serde"] }
 open = "1.1"
+regex = "0.1.80"
```

Comment:
The last version of regex seems to be `0.2.1` is there a reason for using this specific version?